### ISSUE TITLE
debugger traces back to wrong line while digging in react source code

### ISSUE THREAD

**Issue by TomYang1993** [microsoft/vscode-chrome-debug#1004]
- VS Code Version: 1.44.2
- Log file (set `"trace": true` in launch config):

Steps to reproduce:

1. set up any basic react app
2. set break point at line 'return' in render function
3. trace the call stack of 'return' in reactJS source code
4. you will find it's tracing back to wrong line number.
 or maybe the line number is right just the file is formatted differently

As shown below, in debugger it traces back to wrong line. in chrome developer tool, it traces back to correct line in source.

![Screen Shot 2020-04-28 at 11 42 23 PM](https://user-images.githubusercontent.com/20257736/80559048-6890ff00-89aa-11ea-9efd-7d587724b9c1.png)
![Screen Shot 2020-04-28 at 11 42 36 PM](https://user-images.githubusercontent.com/20257736/80559069-76468480-89aa-11ea-9d52-d5ec8de7a7cf.png)
![Screen Shot 2020-04-28 at 11 43 03 PM](https://user-images.githubusercontent.com/20257736/80559088-8199b000-89aa-11ea-9b4b-0253f6348ae3.png)
![Screen Shot 2020-04-28 at 11 43 15 PM](https://user-images.githubusercontent.com/20257736/80559097-8494a080-89aa-11ea-9aa9-457232021699.png)

**Comment by roblourens** [comment]
Could you try this with our new debugger implementation? https://code.visualstudio.com/updates/v1_43#_new-javascript-debugger

**Comment by TomYang1993** [comment]
**launch.json**
`{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "chrome",
            "request": "launch",
            "name": "Launch Chrome against localhost",
            "url": "http://localhost:3000",
            "webRoot": "${workspaceFolder}"
        }
    ]
}`

**settings.json**
`{
    "debug.javascript.usePreview": true,
    "debug.javascript.terminalOptions": {
    
    }
}`

I added the new javascript debugger as extension, uninstalled the chrome one.
It errored out saying
**Configured debug type 'chrome' is not supported.**

**Comment by jasonwilliams** [comment]
Try “pwa-chrome”

**Comment by TomYang1993** [comment]
yup, "pwa-chrome" works, but javascript debugger(nightly) still points to wrong line, same behavior as debugger for chrome

**Comment by roblourens** [comment]
I recommend sticking with pwa-chrome then, since this implementation will be the default soon.

**Comment by jasonwilliams** [comment]
> I recommend sticking with pwa-chrome then, since this implementation will be the default soon.

I think @TomYang1993 meant the new debugger works but he can still reproduce the original issue.

**Comment by roblourens** [comment]
Sorry, misunderstood. Can you set `"trace": true` in your launch config, repro the issue, then upload the log?
